src/controllers/controlpickermenu.cpp

```diff
@@ -631,8 +651,9 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                            tr("Switch to either next or previous effect"),
                            effectSlotMenu, false, slotDescriptionPrefix);
 
-                const int numParameterSlots = static_cast<int>(ControlObject::get(
-                        ConfigKey(effectSlotGroup, "num_parameterslots")));
+                // Effect parameter knobs
+                const int numParameterSlots = ControlObject::get(
```

Comment:
```
src/controllers/controlpickermenu.cpp:655:65: warning: conversion from ‘double’ to ‘int’ may change value [-Wfloat-conversion]
  655 |                 const int numParameterSlots = ControlObject::get(
      |                                               ~~~~~~~~~~~~~~~~~~^
  656 |                         ConfigKey(effectSlotGroup, "num_parameterslots"));
      |                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
so this was the merge conflict..didn't look close enough.
reverted.